Shaders/ReShade.fxh

```diff
@@ -4,6 +4,21 @@
 	#error "ReShade 3.0+ is required to use this header file"
 #endif
 
+#if __RESHADE__ < 30101
+	#define __RESHADE_DEPTH_LINEARIZATION_FAR_PLANE_DEFAULT__ 1000.0
+	#define __RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN_DEFAULT__ 0
+	#define __RESHADE_DEPTH_INPUT_IS_REVERSED_DEFAULT__ 0
+	#define __RESHADE_DEPTH_INPUT_IS_LOGARITHMIC_DEFAULT__ 0
+#else
+	#define __RESHADE_DEPTH_LINEARIZATION_FAR_PLANE_DEFAULT__ 1000.0
```

Comment:
I'd prefer if these were defined in DisplayDepth.fx and not the global ReShade.fxh header.